docs/how-to/h-configure-s3.md

```diff
@@ -0,0 +1,23 @@
+Configuring S3-storage requires that you have a cluster with at least three-nodes deployed and access to S3 storage.  If you don't have a three node replica set read the [Managing Units How-To](/t/charmed-postgresql-tutorial-managing-units/TODO). 
```

Comment:
A three-nodes cluster is a requirement from Mykola? Because PostgreSQL can have one unit and be related to S3 integrator and create backups.

Reply:
Best practyces (and Mykola) wants to have even amount of nodes. One fits :-)
P.S. in this particular case it was a template from MongoDB. Updated to allow one unit "cluster". Tnx!